collections/_article/gdc-2025-retrospective.md

```diff
@@ -0,0 +1,60 @@
+
+<img src="/storage/blog/gdc-2025/godot-talk-after.webp" alt="Emilio Coppola answering questions from multiple talk attendees." />
+
+All of this wouldn't have been possible without the sponsor of the talk, [W4 Games](https://www.w4games.com/). Our many thanks to them.
+
+## Godot meetup
+
+Inspired by the successful [Blender meetup](https://bsky.app/profile/blender.org/post/3lk7d6lbhf22o) at the beginning of the week, we decided to try one on our own. [With only 2 days of notice](https://bsky.app/profile/adamscott.studio/post/3lkopw475z22y), we weren't expecting much.
+
+<img src="/storage/blog/gdc-2025/godot-meetup.webp" alt="Group picture of the Godot meetup participants at the Yerba Buena Gardens in San Francisco during the GDC" />
+
+But the Godot community shared the word and answered! We were stunned by the number of people that showed up.
+
+Those who attended the meetup had the opportunity to talk with regular Godot contributors, Emilio Coppola, and Ariel Manzur (co-founder of the Godot Engine and board member of the [Godot Foundation](https://godot.foundation)).
+
+Our many thanks to all of those who showed up! It was a pleasure to meet all of you. Fun fact: even if I called the end of the meetup at the 2-hour mark, somebody told me that the last people left nearly one hour after.
+
+## Showcased Godot games
+### Kneedle Drop: An Into the Woolderness Game
+
+"A RHYTHM KNITTING GAME?!" That's how [Immortal Funk Games](https://www.immortalfunkgames.com/) describes their spin-off game taylor-made (and chosen!) for the [alt.ctrl.GDC showcase of alternative controllers](https://gdconf.com/alt-ctrl-gdc).
+
+<img src="/storage/blog/gdc-2025/kneedle-drop-gameplay.webp" alt="Kneedle Drop gameplay, where two people hold each two controllers shaped as knitting needles." />
+
+In this game that mimic the essence of knitting as well as Guitar Hero does for playing guitar, either you tap the point of the needles, or you need to tap the left controller on the right one from one side or the other, all while keeping the rhythm.
+
+<img src="/storage/blog/gdc-2025/kneedle-drop-team.webp" alt="The team behind Kneedle Drop, in front of their game showing on a screen." />
+
+The team was explaining to me how nice working with Godot is, especially for artists.
+
+Be sure to check out [Into the Woolderness](https://www.immortalfunkgames.com/into-the-woolderness), the main project of the team. It's a cozy game about crafting and knitting garments for your dogs. Be reassured (or sad), as this one will play on a standard controller.
+
+### Year Unknown
+
+[Year Unknown](https://store.steampowered.com/app/2565520/Year_Unknown/) is a game released in October last year by Julian Heuser. It is a "walking simulator" about exploring a minimalist landscape with the help of two AIs, trying to uncover secrets about humanity.
```

Comment:
The Steam link presumably needs that curator thingy added.

Reply:
It already has.

Reply:
?